src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSenderTest.java

```diff
@@ -106,4 +108,8 @@ private JavaMailSender getMailSender() {
 
         return mailSender;
     }
+
+    private LocalDate getYesterday() {
+        return LocalDate.now().minusDays(1);
+    }
```

Comment:
It's called once.
Isn't
```
LocalDate yesterday = LocalDate.now().minusDays(1);
```
clear?

Reply:
This approach is similar to what I used in https://github.com/hmcts/blob-router-service/pull/598 and I assume it is justified improving readability

Reply:
ok, I will inline this call